draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -99,11 +99,14 @@
                 or answers can be used to indicate whether a new DTLS association is to be established.
             </t>
             <t>
-                The SDP 'tls-id' attribute can also be used for negotiating a TLS connection, using
+                The SDP 'tls-id' attribute can be specified when negotiating a TLS connection, using
                 the procedures in this document in conjunction with the procedures in <xref format="default"
                 pageno="false" target="RFC5763"/> and <xref format="default" pageno="false" target="RFC8122"/>.
-                The TLS specific considerations are described in <xref format="default" pageno="false"
-                target="sec-tls-cons"/>.
+                The unique combination of SDP 'tls-id' attribute values can be used to identity the negotiated

```

Comment:
I wanted to clarify this once again. There are two scenarios when tls-id generated by one end point can be sued for more then one DTLS association:
1. Forking: Single offer with the single tls-id can end up establishing more then one dialog and more then DTLS association which should be differentiated by the tls-id value from the answers

2. DTLS association initiated by the answering party:  offerer wants to continue to run the existing DTLS association and sends an offer with existing tls-id value. Answerer decides that it wants to establish a new DTLS association and  sends a response with new tls-id value. Two DTLS association will only be different by the tls-id value from the answering party.

You can design something that only uses a single ID for DTLS association and works with both of the described scenarios, but this will require a major document redesign which we would like to avoid.